src/sources/kubernetes_logs/mod.rs

```diff
@@ -400,29 +402,34 @@ impl Source {
         let events = events.flatten();
         let events = events.map(move |line| {
             let byte_size = line.text.len();
+            emit!(&BytesReceived {
+                byte_size,
+                protocol: "http",
+            });
+
             let mut event = create_event(
                 line.text,
                 &line.filename,
                 ingestion_timestamp_field.as_deref(),
             );
             let file_info = annotator.annotate(&mut event, &line.filename);
 
-            emit!(&KubernetesLogsEventReceived {
+            emit!(&KubernetesLogsEventsReceived {
                 file: &line.filename,
                 byte_size,
```

Comment:
This `byte_size` isn't right. It should be using `BytesSizeOf::size_of` to calculate in-memory bytes.

Reply:
good catch, I didn't notice it.